src/main/kotlin/com/optravis/jooq/gradle/Config.kt

```diff
@@ -78,7 +90,11 @@ public data class DbConnectionConfig(
     internal val urlTemplate: String,
 ) : Serializable {
     internal companion object {
```

Comment:
Would we need to set the companion object as public too?

Reply:
I don't think it is *needed* per say. But yes, we can and its probably better to do so.